internal/stringify/fmt_test.go

```diff
@@ -88,6 +107,16 @@ func TestFmt(t *testing.T) {
 			input:  []any{1, "a", true, aStringer{"smarf"}},
 			expect: []string{"1", "a", "true", "smarf"},
 		},
+		{
```

Comment:
Could you please add a test case that proves the fmt.Formatter check gets used before the spew?

Reply:
This test proves it works:
```
		{
			name:   "concealer",
			input:  []any{aConcealer{"fisher flannigan fitzbog"}},
			expect: []string{"***"},
		},
```

Also it was the reason to add it :) 
But I will add a separate one